src/Repository/FileRepository.php

```diff
@@ -157,7 +157,7 @@ public function findFilteredPaginated(SearchTerritoryFiles $searchTerritoryFiles
 
         $territory = $territory ?: $searchTerritoryFiles->getTerritory();
         if ($territory) {
-            $qb->andWhere('f.territory = :territory')
+            $qb->andWhere('f.territory = :territory OR f.territory IS NULL')
```

Comment:
à voir si on rajoute le `f.territory is null` pour les SA et les agents multi-partenaires